tests/scripts/all.sh

```diff
@@ -2340,6 +2340,9 @@ component_test_psa_crypto_config_accel_ecdsa () {
     loc_accel_list="ALG_ECDSA ALG_DETERMINISTIC_ECDSA \
                     $(helper_get_psa_key_type_list "ECC")"
 
+    # Note: Those are handled in a special way by the libtestdriver machinery,
```

Comment:
Non-blocker: I don't really understand this special handling. Why are curves treated differently from key types? But since this is a preexisting problem with the complexity of the libtestdriver1 setup, and we have some checks in place (`grep` in the `all.sh` jobs, driver testing parity checks in outcome analysis), I can live with the changes in this pull requests. I'd probably reject more complex changes without some prior rework of the libtestdriver1 setup.

Reply:
I don't understand it either, and I was meaning to create an issue about looking if it can be simplified (once I'm done with release-critical things).

Reply:
#8262